zeroex/order.go

```diff
@@ -173,7 +173,12 @@ func (o *OrderEvent) UnmarshalJSON(data []byte) error {
 }
 
 func (o *OrderEvent) fromOrderEventJSON(orderEventJSON orderEventJSON) error {
-	o.Timestamp = orderEventJSON.Timestamp
+	layout := "2006-01-02 15:04:05 -0700 MST"
```

Comment:
I think you can just use `time.RFC3339`. That's the format used in JSON and supported by both Go and JavaScript.

Reply:
Okay, good to know. I'll try that